### ISSUE TITLE
Mattermost 10.0.0 suggests upgrading to 9.11.1

### ISSUE THREAD

**Issue by jtremesay** [mattermost/mattermost#28156]
#### Summary

Just installed Mattermost Teams edition 10.0.0 on docker (using this [image](https://hub.docker.com/layers/mattermost/mattermost-team-edition/10.0.0/images/sha256-eb72fed23b0581eb17ed082f7926797716612594dcd1a9aac2537efcddc0af56?context=explore)).
The "Workspace Optimization" page tells me to update to 9.11.1.

#### Steps to reproduce

- Install Mattermost 10.0.0.0
- Go to system console > reporting > workspace optmization
- Look at the proposed updates

#### Operating System

- official docker container on a Ubuntu 22.04 LTS server

#### Mattermost Server Version

```
Mattermost Version: 10.0.0
Database Schema Version: 125
Build Number: 10671348655
Database: postgres
```

#### Expected behavior

I'm pretty sure that 10.0.0 > 9.11.1 and mattermost should not offer me this update

#### Additional Information

![2024-09-11-092428_638x580_scrot](https://github.com/user-attachments/assets/67b1262b-698b-4641-82e5-d67434af608d)

![2024-09-11-092503_1141x647_scrot](https://github.com/user-attachments/assets/e7dc3849-1c1d-4f1a-902c-d9b73d2e7aac)

**Comment by amyblais** [comment]
This will get resolved on Sept 16th, which is when v10.0 will be officially published/released.

**Comment by amyblais** [comment]
https://mattermost.atlassian.net/browse/MM-60516
